packages/edit-site/src/components/editor/index.js

```diff
@@ -176,11 +177,18 @@ export default function Editor( { listViewToggleElement, isLoading } ) {
 			POST_TYPE_LABELS[ editedPostType ] ??
 				POST_TYPE_LABELS[ TEMPLATE_POST_TYPE ]
 		);
+		accessibleTitle = sprintf(
```

Comment:
Instead of using this new `accessibleTitle` string, which is the same as `title` with the ` ‹ Editor` part stripped out, I'd strip out ` ‹ Editor` from the existing title and add it later in `useTitle`. I think we could entirely avoid to use `accessibleTitle`?